src/app/hearing-part/components/listing-create/listing-create.component.ts

```diff
@@ -290,13 +291,15 @@ export class ListingCreateComponent implements OnInit {
                 this.listing.hearing.duration = moment.duration(
                     moment.duration(this.asDaysPipe.transform(this.listing.hearing.duration), 'days').asMinutes()
                     , 'minutes');
+                this.listing.hearing.isMultiSession = true;
```

Comment:
Maybe it would be good to use this instead of the whole ListingType? and then set this field in setFormGroup - just a thought

Reply:
Done